pkg/workload/tpcc/partition.go

```diff
@@ -262,7 +262,7 @@ func configureZone(db *gosql.DB, cfg zoneConfig, table, partition string, partId
 		opts = fmt.Sprintf(`constraints = '[+%s]'`, kv)
```

Comment:
We also need it here, right?

Reply:
No, because the constraints here aren't per replica, so we don't need to set num_replicas

Reply:
👍